OmniBLE/PumpManagerUI/Views/DeliveryUncertaintyRecoveryView.swift

```diff
@@ -19,16 +19,23 @@ struct DeliveryUncertaintyRecoveryView: View {
 
     var body: some View {
         GuidePage(content: {
-            Text(String(format: LocalizedString("%1$@ has been unable to communicate with the pod on your body since %2$@.\n\nWithout communication with the pod, the app cannot continue to send commands for insulin delivery or display accurate, recent information about your active insulin or the insulin being delivered by the Pod.\n\nMonitor your glucose closely for the next 6 or more hours, as there may or may not be insulin actively working in your body that %3$@ cannot display.", comment: "Format string for main text of delivery uncertainty recovery page. (1: app name)(2: date of command)(3: app name)"), self.model.appName, self.uncertaintyDateLocalizedString, self.model.appName))
+            Text(String(format: LocalizedString("%1$@ has been unable to communicate with the pod on your body since %2$@.\n\nDo not Discard the Pod without scrolling to read this entire screen.\n\nCommunication was interrupted at a critical time. Information about active insulin / pod temp basal rate is uncertain.\n\nYou can toggle phone Bluetooth off and on or quit and restart the app. (Wait up to 30 seconds for recovery.)\n\nIf you decide to give up because communication cannot be restored, tap the Discard Pod button, follow the steps and start a new pod.\nMonitor your glucose closely for the next 6 hours, as there may or may not be insulin actively working in your body that %3$@ cannot display.", comment: "Format string for main text of delivery uncertainty recovery page. (1: app name)(2: date of command)(3: app name)"), self.model.appName, self.uncertaintyDateLocalizedString, self.model.appName))
                 .padding([.top, .bottom])
         }) {
             VStack {
                 Text(LocalizedString("Attemping to re-establish communication", comment: "Description string above progress indicator while attempting to re-establish communication from an unacknowledged command")).padding(.top)
                 ProgressIndicatorView(state: .indeterminantProgress)
+                Button(action: {
+                    self.model.onDismiss?()
+                }) {
+                    Text(LocalizedString("Keep Waiting", comment: "Button title to return to prior screen"))
+                    .actionButtonStyle(.primary)
+                    .padding([.horizontal])
+                }
                 Button(action: {
                     self.model.podDeactivationChosen()
                 }) {
-                    Text(LocalizedString("Deactivate Pod", comment: "Button title to deactive pod on uncertain program"))
+                    Text(LocalizedString("Discard Pod", comment: "Button title to discard pod on uncertain program"))
```

Comment:
I think this still takes the user to deactivate, rather than discard, but I'm on the fence about which is better.

Reply:
Yes it does because I couldn’t figure out how to change it. 

But on the other hand, if it succeeds in reaching the pod, the pod gets deactivated so there’s no screamer but at the same time you’re mad because you gave up too soon.